packages/hooks/src/useTokenMetadata.ts

```
@@ -1,175 +1,51 @@
-    data: result.data
-      ? ({
-          address: address as Hex,
-          decimals: Number(result.data[0]),
-          name: result.data[1],
-          symbol: result.data[2],
-          totalSupply: result.data[3],
-        } as TokenMetadata)
-      : undefined,
-  };
-};
+import { fetchTokenMetadata } from './helpers';
```

ESLint

[error] Missing file extension for "./helpers"

(import/extensions)